Gemfile

```diff
@@ -0,0 +1,50 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.2'
+
```

Comment:
[OPTIONAL]
Swagger is good option for documentation, it'd make your API a bit more interactive. See [rswag gem](https://github.com/rswag/rswag)